go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/multiformats/go-multihash
+module github.com/laudiacay/go-multihash
```

Comment:
You shouldn't be renaming the module, even for testing this is going to cause you a boatload of problems. You can use go mod replace directive to help you with imports though (e.g. add the line `replace github.com/multiformats/go-multihash => github.com/laudiacay/go-multihash <commit hash>`)

Reply:
it won't let me do that and import without problems into kubo? It gets mad that the module name doesn't match the path. It works ok and lets me run `go vet` when I change the name. I'm sorry about all the back and forth, I just don't know what I'm doing with go imports.

Reply:
@laudiacay `replace` check against the original name of the module, not the replaced one.

So if I do
```
replace github.com/multiformats/go-multihash => github.com/laudiacay/go-multihash version
```
`github.com/laudiacay/go-multihash`'s module name need to be `github.com/multiformats/go-multihash`.

Also this is temporary while the PRs are in flight to see CI pass, whoever is merging this is gonna make new versions, remove the replaces with proper version bumps.